nemo/collections/asr/parts/mixins/transcription.py

```diff
@@ -56,7 +57,7 @@ class InternalTranscribeConfig:
 @dataclass
 class TranscribeConfig:
     batch_size: int = 4
-    return_hypotheses: bool = False
+    return_all_hypotheses: bool = False
```

Comment:
Don't make this pedantic change please @nithinraok

Reply:
Lets revert this @Ssofja but keep `timestep` to `timestamp` change.